addons/html_editor/static/tests/syntax_highlighting.test.js

```diff
@@ -33,36 +39,43 @@ const changeLanguage = async (textarea, from, to) => {
     await click(`.o_language_selector .o-dropdown-item[name='${to}']`);
     // Code Toolbar should show the new language name.
     await waitFor(`.o_code_toolbar button[name='language'][title='${to}']`);
+    await animationFrame(); // wait for effect
 };
 
 const configWithEmbeddings = {
     Plugins: [...MAIN_PLUGINS, ...EMBEDDED_COMPONENT_PLUGINS],
     resources: { embedded_components: MAIN_EMBEDDINGS },
 };
+const testEditorWithHighlightedContent = async (config) =>
+    await testEditor({
+        ...config,
+        compareFunction: compareHighlightedContent,
+        config: configWithEmbeddings,
+    });
 
 beforeEach(patchPrism);
 
 test("starting edition with a pre activates syntax highlighting", async () => {
-    await testEditor({
-        compareFunction: compareHighlightedContent,
+    await testEditorWithHighlightedContent({
         contentBefore: "<pre>some code</pre>",
         contentBeforeEdit:
             '<p data-selection-placeholder=""><br></p>' +
             highlightedPre({ value: "some code" }) +
             '<p data-selection-placeholder="" style="margin: -9px 0px 8px;"><br></p>',
-        stepFunction: async () => press(["ctrl", "z"]),
+        stepFunction: async () => {
+            await press(["ctrl", "z"]);
+            await animationFrame();
```

Comment:
Why do you add an `await animationFrame();` after the new `press()` utils method already does one ?

I would think we probably don't need two.